backToCart/index.ts

```diff
@@ -26,7 +25,7 @@ export default class BackToCart {
     getCatalogueUrl(): string {
         if (!this.template) {
             console.warn('<Back to Cart> Trying to get URL before template is set');
-            return '';
+            return;
```

Comment:
Method declares a return value of `string` but it seems this returns `undefined`?

Reply:
switched to have both return values, value of undefined used to cancel button click callback